sdf/quickstart.mdx

```diff
@@ -9,16 +9,15 @@ Provisioning and operating a Stateful Dataflow requires the following system com
 
 1. [Fluvio Cluster] to enable dataflows to consume and produce streaming data.
```

Comment:
it this resolving?

Reply:
Yeah! Link here: https://360843c5.fluvio-docs-15u.pages.dev/sdf/quickstart

Goes here: https://360843c5.fluvio-docs-15u.pages.dev/docs/fluvio/quickstart#start-a-cluster